TheMET/TheMET/Network/LoadingImageView.swift

```diff
@@ -45,7 +45,6 @@ class LoadingImageView: UIView {
         self.addSubview(self.imageView)
         self.addSubview(self.activityIndicator)
         self.addSubview(self.failedImageView)
-        self.imageView.contentMode = .scaleAspectFit
```

Comment:
Why `self.imageView.contentMode` is not set anymore?

Reply:
fixed